packages/grid/_modules_/grid/components/GridCheckboxRenderer.tsx

```diff
@@ -4,11 +4,11 @@ import { useGridSelector } from '../hooks/features/core/useGridSelector';
 import { visibleSortedGridRowsSelector } from '../hooks/features/filter/gridFilterSelector';
 import { gridRowCountSelector } from '../hooks/features/rows/gridRowsSelector';
 import { selectedGridRowsCountSelector } from '../hooks/features/selection/gridSelectionSelector';
-import { GridColParams } from '../models/params/gridColParams';
+import { GridColumnHeaderParams } from '../models/params/gridColumnHeaderParams';
 import { GridCellParams } from '../models/params/gridCellParams';
 import { GridApiContext } from './GridApiContext';
 
-export const GridHeaderCheckbox: React.FC<GridColParams> = () => {
+export const GridHeaderCheckbox: React.FC<GridColumnHeaderParams> = () => {
```

Comment:
A note for the future self (outside of the scope of this PR) and Matheus, `React.FC` is wrong, the component has no `children` prop

Reply:
True, another PR to update all components that don't use `children` would be good.

Reply:
@m4theushw Do you want to open an issue for keeping track of it? I'm not saying a priority but, that it would be great to handle it, at some point. Some of these components are public, hence hurt the developers using them.

Reply:
https://github.com/mui-org/material-ui-x/issues/1325